### ISSUE TITLE
"Second next turn" widget is colored incorrectly

### ISSUE THREAD

**Issue by MagielBruntink** [osmandapp/OsmAnd-iOS#3054]
During navigation, the "second next turn widget" color (gray/yellow/green) is incorrectly based on the further distance from the next turn. The expected behavior is that the widget color is based on the distance of the user to the second next turn. 

In the screenshot the "second next turn" widget is green while the next turn is still 928 meters away. In this case, both widgets should be yellow.

![image](https://github.com/osmandapp/OsmAnd-iOS/assets/3635696/01b75938-0ea9-45e7-96e1-3e524a91c27e)

**Comment by MagielBruntink** [comment]
I understand the reasoning. What I thought is that you should bring the second next turn into attention if the first next turn is also very close.

**Comment by sonora** [comment]
Yes, the color of the second turn is relative to the first turn, or else it would rarely be green to indicate the two next turns are very close. I would not change or long-used behavior.
